src/stemmabench/algorithms/utils.py

```diff
@@ -18,6 +19,19 @@ def load_text(path_to_text: str) -> str:
         with open(path_to_text, "r", encoding="utf-8") as file:
             return file.read()
 
+    @staticmethod
+    def get_text_list(folder_path: str) -> List[str]:
+        """For a given folder path returns a list of all the text names in that folder.
+        Will remove all names that contain the subsring "edge" from the list.
+        
+        ### Args:
+            - folder_path (str): The path to the folder that containing stemma texts.
+
+        ### Returns:
+            - list: List of manuscript names.
+        """
+        return [l.replace(".txt","") for l in listdir(folder_path) if l.find("edge") == -1]
```

Comment:
There a lot of nicer ways to do this, look into Pahtlib features.

For example : l.stem will give you the filename,  ... and you can glob from the start using .glob() (https://docs.python.org/3/library/pathlib.html):

```suggestion
        return [l.stem for l in Path(folder_path).glob("*.txt") if not "edge" in l.stem]
```

Reply:
Done